client/my-sites/stats/all-time/index.jsx

```diff
@@ -97,27 +97,35 @@ module.exports = React.createClass( {
 						<p>{ this.translate( 'These are your site\'s overall total number of Posts, Views and Visitors as well as the day when you had the most number of Views.' ) }</p>
 					</div>
 
-				<ul className="stats-all-time__list">
-					<li className="stats-all-time__list-item stats-all-time__item-posts">
-						<Gridicon icon="posts" size={ 18 } />
-						<span className="stats-all-time__label">{ this.translate( 'Posts' ) }</span>
-						<span className={ classNames( valueClass, { 'is-low': this.isLow( this.props.allTimeList.response.posts ) } ) }>{ this.ensureValue( this.props.allTimeList.response.posts ) }</span>
+				<ul className="module-tabs">
+					<li className="module-tab">
```

Comment:
@dmsnell I don't think we need to add them as there isn't any styling tied specifically to those classes. Only `is-best` has properties.